cinemy/cinema/views.py

```diff
@@ -1,3 +1,14 @@
 from django.shortcuts import render
+from ratelimit.decorators import ratelimit
+from django.contrib.auth.decorators import login_required
+from .models import Cinema
 
 # Create your views here.
+
+
+@login_required()
+@ratelimit(key="ip", rate="30/m", block=True)
+def select_cinema(request):
+    cinemas = Cinema.objects.all()
```

Comment:
include atribuirea asta in if ul de GET

Reply:
Obiectele vor fi folosite si pe ramura verbului POST dar for now le pot pune strict sub GET